Tests iOS/SignOutTests.swift

```diff
@@ -58,10 +58,12 @@ class SignOutTests: XCTestCase {
         let cellCount = account.cells.count
         XCTAssertTrue(cellCount > 0)
 
-        let logoutRowTappedEvent = await snowplowMicro.getFirstEvent(with: "global-nav.settings.logout")
+        let events = await [snowplowMicro.getFirstEvent(with: "global-nav.settings.logout"), snowplowMicro.getFirstEvent(with: "global-nav.settings.logout-confirmed")]
```

Comment:
is this for organizational purposes and not necessarily performance?

Reply:
@cyndichin performance! `getFirstEvent` makes an http call. By putting them in an array and awaiting the array it will perform the calls on seperate threads. Otherwise it has to work syncronously.